R/epinowcast.R

```diff
@@ -162,12 +162,12 @@ epinowcast <- function(data,
     enw_priors_as_data_list(priors)
   )
 
-  if (!missing$formula %in% "~0") {
+  if (!missing$formula == "~0") {
     warning("The missing data model is highly experimental. There is a
      significant likelihood that bugs are present in its implementation.")
   }
 
-  if (!missing$formula %in% "~0") {
+  if (!missing$formula == "~0") {
     warning("The missing data model is highly experimental. There is a
      significant likelihood that bugs are present in its implementation.")
   }
```

Comment:
this seems oddly repetitive - am I missing something?

Reply:
Yes, this is a duplication and no I don't know why it is in there!